src/test/crimson/test_alien_echo.cc

```diff
@@ -180,11 +183,12 @@ seastar_echo(const entity_addr_t addr, echo_role role, unsigned count)
       server.msgr->set_auth_client(&server.dummy_auth);
       server.msgr->set_auth_server(&server.dummy_auth);
       return server.msgr->bind(entity_addrvec_t{addr}
-      ).then([&server] {
-	auto chained_dispatchers = seastar::make_lw_shared<ChainedDispatchers>();
-	chained_dispatchers->push_back(server.dispatcher);
-        return server.msgr->start(chained_dispatchers);
-      }).then([&dispatcher=server.dispatcher, count] {
+      ).safe_then([&server] {
+        return server.msgr->start({&server.dispatcher});
+      }, crimson::net::Messenger::bind_ertr::all_same_way([](auto& e) {
+        std::cerr << "bind failed" << std::endl;
+        ceph_abort();
```

Comment:
How about `ceph_abort_msg("bind failed")`?

Reply:
@rzarzynski fixed and repushed.